src/getHumanAge.test.js

```diff
@@ -7,4 +7,51 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  it('should return an array with 2 elements', () => {
+    expect(getHumanAge(0, 0))
+      .toHaveLength(2);
+  });
+
+  it(`'should return an array of 2 zero
```

Comment:
You should use only one kind of quotes, not ` 'like this' `.

Reply:
Thanks. Edited.